systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeCorsST.java

```diff
@@ -43,12 +43,11 @@
 public class HttpBridgeCorsST extends AbstractST {
 
     private static final Logger LOGGER = LogManager.getLogger(HttpBridgeCorsST.class);
-    private static final String NAMESPACE = Constants.BRIDGE_KAFKA_CORS_NAMESPACE;
-    private final String httpBridgeCorsClusterName = "http-bridge-cors-cluster-name";
-
     private static final String ALLOWED_ORIGIN = "https://strimzi.io";
     private static final String NOT_ALLOWED_ORIGIN = "https://evil.io";
 
+    private final String httpBridgeCorsClusterName = "http-bridge-cors-cluster-name";
+    private final String namespace = testSuiteNamespaceManager.getMapOfAdditionalNamespaces().get(HttpBridgeCorsST.class.getSimpleName()).stream().findFirst().get();
```

Comment:
If you don't like this piece of code `.stream().findFirst().get()` it's because in the `HttpBridgeCorsST` we are using only one namespace so that's why that code is so long...Or you don't like querying map with get `getMapOfAdditionalNamespaces().get(...)?

Reply:
Yeah sry, I forgot you thinking about future use case of multiple namespaces for one test suite. You can ignore this 😄